FAF99301 Extra Life Plando.pnach

```diff
@@ -622,10 +622,8 @@ patch=1,EE,11C583E4,extended,00000134 // Dragoon
 
 //Boss Replacement
 // Marluxia Safety
-patch=1,EE,E004050C,extended,0032BAE0
-patch=1,EE,E0010166,extended,01CE0B6A
+patch=1,EE,E002050C,extended,0032BAE0
 patch=1,EE,11CE0B6A,extended,0000005C // Revert Dolan from Marluxia battle
-patch=1,EE,E0010166,extended,01CE0B6C
 patch=1,EE,11CE0B6C,extended,0000005D // Revert Goffy from Marluxia battle

```

Comment:
This is Marluxia's safety lines. Haven't been tested though.

Reply:
tested with both dying to Marluxia and then clearing the fight, no crashes or issues to note